packages/odata/queryable.ts

```diff
@@ -13,6 +13,13 @@ import { PipelineMethod } from "./pipeline";
 import { IODataParser, ODataParser } from "./parsers";
 
 export function cloneQueryableData(source: Partial<IQueryableData>): Partial<IQueryableData> {
```

Comment:
I couldn't find tests for this. 
I could write a test covering the cases with body but I'd need at least a stub with all the setup.